### ISSUE TITLE
Motion Senor Icon in Chip Cards

### ISSUE THREAD

**Issue by declankenny** [piitaya/lovelace-mushroom#87]
Currently, when a motion sensor entity is added to a chip card, it defaults to a `checkbox-marked-circle` icon. The out of box icon for a motion sensor depends on its state, using either `motion-detector` or `motion-sensor-off`. It would be great if this was also the default behavior in Mushroom

**Comment by piitaya** [comment]
We currently use custom-cards for the icon definition : https://github.com/custom-cards/custom-card-helpers/blob/master/src/domain_icons.ts but it's not updated with latest icon rules based on icon domain AND device_class.

**Comment by declankenny** [comment]
I am happy to move the issue over to the custom card helpers repo. Paul, I think the correct approach here is that binary sensor domain should look to the [device class](https://www.home-assistant.io/integrations/binary_sensor/#device-class) to determine the correct icon.

**Comment by declankenny** [comment]
Added an issue in the upstream library
https://github.com/custom-cards/custom-card-helpers/issues/57

**Comment by mochoandre** [comment]
Yup!! Same Problem here. I am not a developer, but if it will look for the device class to determine the correct icon, the icons will loose previous customizations right (like in customize.yaml)?
